nncf/__init__.py

```diff
@@ -22,28 +22,39 @@
 from nncf.quantization import quantize
 from nncf.quantization import quantize_with_accuracy_control
 
+_LOADED_FRAMEWORKS = {
+    "torch": True,
+    "tensorflow": True,
+    "onnx": True,
+    "openvino": True
+}
+
 try:
     import torch
 except ImportError:
-    torch = None
+    _LOADED_FRAMEWORKS["torch"] = False
 
 try:
     import tensorflow as tf
 except ImportError:
-    tf = None
+    _LOADED_FRAMEWORKS["tensorflow"] = False
 
 try:
     import onnx
 except ImportError:
-    onnx = None
+    _LOADED_FRAMEWORKS["onnx"] = False
 
 try:
     import openvino.runtime as ov_runtime
     import openvino.tools.pot as ov_pot
 except ImportError:
-    ov_runtime = None
-    ov_pot = None
+    _LOADED_FRAMEWORKS["openvino"] = False
```

Comment:
We have both `openvino` and `openvino_native`.
cc @l-bat

Reply:
As confirmed on the sync today, we will only have "openvino" and the native/POT distinction will be abstracted away.